kubecontrolplane/v1/types.go

```diff
@@ -62,6 +62,11 @@ type KubeAPIServerConfig struct {
 
 	// TODO this needs to be removed.
 	APIServerArguments map[string]Arguments `json:"apiServerArguments"`
+
+	// MinimumKubeletVersion is the lowest version of a kubelet that can meaningfully join the cluster.
```

Comment:
`s/can meaningfully join/is allowed to join/`

Reply:
it's allowed to join but it's blocked from doing anything meaningful in the cluster.